superset/data/birth_names.py

```diff
@@ -64,15 +65,18 @@ def load_birth_names():
     obj.filter_select_enabled = True
 
     if not any(col.column_name == 'num_california' for col in obj.columns):
+        col_state = str(column('state').compile(db.engine))
```

Comment:
@villebro is this approach favored over using SQLAlchemy's [quote](https://docs.sqlalchemy.org/en/13/core/internals.html?highlight=dialect#sqlalchemy.sql.compiler.IdentifierPreparer.quote) function? Asking to learn.

Reply:
I believe `sqlalchemy.sql.compiler.IdentifierPreparer` and it's method `quote` is a slightly lower level construct, and I was personally unable to get it to work properly. In my experience it is recommended to always use SQLAlchemy factory functions when possible; same reason why columns are usually made by calling `sqlalchemy.sql.column`, not directly creating an instance of `sqlalchemy.sql.elements.ColumnClause`, which is what `column` ultimately returns. Personally I find this slightly confusing, but I believe the design is made to decouple the implementation code from the inner workings of SQLAlchemy.